Models/TrainingProgram.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace BangazonAPI.Models
+{
+    public class TrainingProgram
+    {
+        [Key]
+        public int TrainingProgramId { get; set; }
+
+        [StringLength(80,
+            ErrorMessage ="Too Many Characters"), Required]
+        public string ProgramName { get; set; }
+  
+        [Required]
+        [DataType(DataType.Date)]
+        [DatabaseGenerated(DatabaseGeneratedOption.Computed)]
+        public DateTime StartDate { get; set; }
+
+        [Required]
+        [DataType(DataType.Date)]
+        public DateTime EndDate { get; set; }
+
+        [Required]
+        public int MaximumAttendees { get; set; }
```

Comment:
From ticket #9: "Employees who signed up for a training program should be included in the response." Let's go ahead and add a collection to hold those employees.